packages/mux-player/src/template.ts

```diff
@@ -14,16 +14,54 @@ import { i18n, stylePropsToString } from './utils';
 import type { MuxTemplateProps } from './types';
 import { StreamTypes } from '@mux/playback-core';
 
+const controlsListStyles: Record<string, string> = {
```

Comment:
thought(non-blocker): I believe we should be able to make this non-dynamic via CSS + list-inclusion attribute selectors (`[controlslist~=notop]`). This shouldn't block the merge, though.